x-pack/plugins/watcher/public/sections/watch_edit/components/watch_edit_detail/watch_edit_detail.html

```diff
@@ -21,24 +21,24 @@
     <div
       class="kuiInputNote kuiInputNote--danger"
       ng-show="watchEditDetail.form.id.$error.pattern"
-    >
-      ID must begin with a letter or underscore and contain only letters, underscores, dashes, and numbers.
-    </div>
+      i18n-id="xpack.watcher.sections.watchEdit.detail.idInputDescription"
+      i18n-default-message="ID must begin with a letter or underscore and contain only letters, underscores, dashes, and numbers."
+    ></div>
     <div
       class="kuiInputNote kuiInputNote--danger"
       ng-show="watchEditDetail.form.id.$touched && watchEditDetail.form.id.$error.required"
-    >
-      ID is required.
-    </div>
+      i18n-id="xpack.watcher.sections.watchEdit.detail.idIsRequiredValidationMessage"
+      i18n-default-message="ID is required."
+    ></div>
   </div>
 
   <div class="kuiFormSection">
     <label
       for="name"
       class="kuiFormLabel"
-    >
-      Name
-    </label>
+      i18n-id="xpack.watcher.sections.watchEdit.detail.nameInputLabel"
```

Comment:
same as above. if you call somewhere labels like this nameLabel, call it the same everywhere